include/swift/AST/DiagnosticsSema.def

```diff
@@ -1725,6 +1725,11 @@ WARNING(redundant_conformance_adhoc_conditional,none,
 NOTE(redundant_conformance_witness_ignored,none,
      "%0 %1 will not be used to satisfy the conformance to %2",
      (DescriptiveDeclKind, DeclName, DeclName))
+WARNING(redundant_conformance_any,none,
+        "redundant conformance of %0 to 'Any'", (Type))
+NOTE(all_types_implicitly_conform_to_any,none,
+     "all types implicitly conform to 'Any'", ())
+
```

Comment:
An alternative phrasing for the note would be "all types are subtypes of 'Any'", which is more technically correct in that `Any` isn't actually a protocol you can conform to (just an empty protocol composition) – but I feel the current wording is more user friendly. Thoughts?